src/main/java/frc/robot/RobotContainer.java

```diff
@@ -13,14 +13,21 @@
 import edu.wpi.first.wpilibj.Filesystem;
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
+<<<<<<< HEAD

```

Comment:
Github made you commit merge conflicts? Have you resolved these or are they just on the branch?

Reply:
By the way, you should keep `HEAD` for all of these. The conflicts look to be from very old changes from before pathfollowing was merged.